### PR TITLE
Add Battery Level for Fineoffset WH0290 Wireless Air Quality Monitor 

### PR DESCRIPTION
**Pull request by peteakalad**
Adding battery level for Fineoffset WH0290 Air Quality Sensor.  Receiver shows bars for sensor battery level out of 5.  Identified the bits for this and converted to 0 (no bars) to 1.0 (5 bars) so max 1.0 in line with other devices.  Updated packet notes.  Updated device notes - decoder also works for the same sensor branded as Misol PM2.5.  Tested and correct decoded values for battery from 0 (=0 bars) to 1.0 (=5 bars).

### CODE REVIEW

File: src/devices/fineoffset.c [hunk 1]
```diff
@@ -380,12 +387,14 @@ static int fineoffset_WH0290_callback(r_device *decoder, bitbuffer_t *bitbuffer)
...
     int pm100         = (b[4] & 0x3f) << 8 | b[5];
-
+    int battery_bars  = (b[2] & 0x40) >> 4 | (b[4] & 0xC0) >> 6; //out of 5
+    float battery_ok  = battery_bars / 5.0; //convert out of 5 bars to 0 (0 bars) to 1 (5 bars)
 
     /* clang-format off */
     data = data_make(
             "model",            "",             DATA_STRING, _X("Fineoffset-WH0290","Fine Offset Electronics, WH0290"),
             "id",               "ID",           DATA_INT,    id,
+            "battery",          "Battery Level",  DATA_FORMAT, "%.1f", DATA_DOUBLE, battery_ok,
```
Comment (zuckschwerdt) [inline src/devices/fineoffset.c:397]: should be "battery_ok", "battery" is assumed to be "OK" / "LOW"

File: src/devices/fineoffset.c [hunk 2]
```diff
@@ -380,12 +387,14 @@ static int fineoffset_WH0290_callback(r_device *decoder, bitbuffer_t *bitbuffer)
     uint8_t id        = b[1];
     int pm25          = (b[2] & 0x3f) << 8 | b[3];
     int pm100         = (b[4] & 0x3f) << 8 | b[5];
-
+    int battery_bars  = (b[2] & 0x40) >> 4 | (b[4] & 0xC0) >> 6; //out of 5
+    float battery_ok  = battery_bars / 5.0; //convert out of 5 bars to 0 (0 bars) to 1 (5 bars)
```
Comment (zuckschwerdt) [inline src/devices/fineoffset.c:391]: `* 0.2f` is preferred

Comment (peteakalad) [reply src/devices/fineoffset.c:391]: Fix committed.

File: src/devices/fineoffset.c (hunk 1 above)
Comment (peteakalad) [reply src/devices/fineoffset.c]: Fix committed.

File: src/devices/fineoffset.c (hunk 2 above)
Comment (zuckschwerdt) [reply src/devices/fineoffset.c:391]: sorry, I meant `float battery_ok  = battery_bars * 0.2f` the format as `%.1f` is ok ;)

Comment (peteakalad) [reply src/devices/fineoffset.c:391]: my mistake I'll fix

Comment (peteakalad) [reply src/devices/fineoffset.c]: Commited Fix - reverted the format change, back to `%.1f` and changed the `/ 5.0` to `* 0.2f`
Rebuilt and retested successfully after these changes.

**Pull request merged**